app/models/user.rb

```diff
@@ -343,6 +343,10 @@ def photo_path(size = :medium)
   def first_time_poster
     notes.where(status: 1).count == 0
   end
+  
+  def first_time_commenter
+    Comment.where(status: 1).count == 0
```

Comment:
This line is wrong and line is updated in #4098 to `Comment.where(status: 1, uid: uid).count == 0`.